scripts/submit-state-diff.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# Submit payload to Celestia
+# Uses a light node and pre-funded account to submit data
+# to Celestia
+
+# Constants
+NETWORK="mocha"
+AUTH_TOKEN=$(celestia light auth admin --p2p.network $NETWORK)
+VERBOSE=false
+
+# Optional
+NAMESPACE="SN_APP"
+DATA_PATH="src/tests/data/os_output.json"
+
+usage() {
+  echo "usage: $0 [-n namespace] [-d path] [-v]"
+  echo -e "\t submit data to celestia"
+  echo
+  echo -e "\t-n, --namespace                   celestia Namespace(default: SN_APP)"
+  echo -e "\t-d, --data-path                   local path to data for submission"
+}
+
+# Transform long options to short ones
+for arg in "$@"; do
+  shift
+  case "$arg" in
+    "--namespace") set -- "$@" "-n" ;;
+    "--data-path") set -- "$@" "-d" ;;
+    *) set -- "$@" "$arg"
+  esac
+done
+
+# Parse command line arguments
+while getopts ":h:v:n:d:" opt; do
+  case ${opt} in
+    h )
+      usage
+      exit 0
+      ;;
+    n )
+      NAMESPACE="${OPTARG}"
+      ;;
+    d )
+      DATA_PATH="${OPTARG}"
+      ;;
+
+    * )
+      echo "Invalid option: $OPTARG" 1>&2
+      usage
+      exit 1
+      ;;
+  esac
+done
+
+# Check Dependencies
+if ! command -v celestia > /dev/null; then
+    echo "please install:\n\thttps://docs.celestia.org/developers/node-tutorial"
```

Comment:
```suggestion
    echo -e "please install:\n\thttps://docs.celestia.org/developers/node-tutorial"
```

My terminal was outputting "\n\t" without creating newline and tab

Reply:
added